app/src/main/kotlin/com/wire/android/notification/WireNotificationManager.kt

```diff
@@ -71,6 +93,38 @@ class WireNotificationManager @Inject constructor(
         }
 
 
+    /**
+     * Infinitely listen for the new IncomingCalls, notify about it and do additional actions if needed.
+     * Can be used for listening for the Notifications when the app is running.
+     * @param userIdFlow Flow of QualifiedID of User
```

Comment:
Is this calling or receiving user? I assume the first. If so can we rename it accordingly?
```suggestion
     * @param userIdFlow Flow of QualifiedID of Calling User
```

Reply:
no, it's a receiver user. As far as we gonna support multiple users FCM provides us receiver user, so we know which user is called.

Reply:
Should we then rename to `selfUserIdFlow` or `receiverUserIdFlow`?

Reply:
We agreed in the collective to use prefix `observe` for naming if it's a flow.
So maybe `observeUserId` ?